packages/text-document-utils/src/text-formatting.ts

```diff
@@ -0,0 +1,36 @@
+import type { DocumentUri } from 'vscode-languageserver-textdocument';
+import { TextDocument } from 'vscode-languageserver-textdocument';
+
+export const wrapInQuotes = (text: string): string => `'${text}'`;
+
+export interface FormatterOptions {
+    /**
+     * Indent lines with tabs instead of spaces
+     *
+     * @default false
+     */
+    useTabs: boolean;
+    /**
+     * Specify the number of spaces per indentation-level.
+     */
+    tabWidth: number;
+    /**
+     * Specify the line length that the printer will wrap on.
+     */
+    printWidth: number;
+    /**
+     * @default true
+     */
+    useSnippetSyntax: boolean;
+}
+
+export const printOptions: FormatterOptions = {
+    printWidth: 300,
+    tabWidth: 4,
+    useTabs: false,
+    useSnippetSyntax: true
+};
+
+export const createTextDocument = (uri: DocumentUri, languageId: string, version: number, content: string) => {
```

Comment:
Whats the purpose of wrapping already existing factory function?

Reply:
removed, had issue while importing in the other module before, but that was due to incorrect type handling

Reply:
I can still see it here.